include/utils/algorithm.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <algorithm>
+
+#include "common.hpp"
+
+POLYBAR_NS
+
+namespace algo_util {
```

Comment:
Did you add this, only so you could use binary search instead of linear seach when replacing tokens?

If so, please remove this.

In the common case the token whitelist will contain 0-2 items, here binary search is really not any faster than linear search.

Reply:
Yeah, I agree, I will remove that